hr_payroll_commission/tests/test_commission.py

```diff
@@ -0,0 +1,130 @@
+# -*- coding: utf-8 -*-
+# © 2016 Coninckx David (Open Net Sarl)
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl).
+
+import datetime
+import openerp.tests.common as common
+
+
+class TestCommission(common.TransactionCase):
+
+    def setUp(self):
+        super(TestCommission, self).setUp()
+
+        self.main_company = self.env.ref('base.main_company')
+        self.main_company.currency_id = self.env.ref("base.CHF").id
+        self.sale_user = self.env['res.users'].with_context(
+            {'no_reset_password': True}
+        ).create(
+            {
+                'name': "Salesman",
+                'company_id': self.main_company.id,
+                'login': "sale",
+                'email': "sale@yourcompany.com",
+            }
+        )
+
+        self.employee = self.env['hr.employee'].create(
+            {
+                'name': 'TEST',
+                'user_id': self.sale_user.id
+            }
+        )
+        self.partner = self.env.ref("base.res_partner_2")
+        self.product = self.env.ref("product.product_product_4")
+        self.account_receivable = self.env['account.account'].search(
```

Comment:
I think you can directly use self.env.ref with l10n_generic_coa.conf_a_recv since at demo it will install the generic chart of accounts...